### DISCUSSION TITLE
Engine power reduced v.6

### DISCUSSION THREAD

**Discussion by Phantomflyer81** [flybywiresim/aircraft#4464] [Help] [answered]
Can anyone tell me why since updating to v6 stable, I canot climb above FL 340 without stall? It's taking ages to get up there. Even with 10% load of pax and baggage it struggles to get up there. Managed power is no go. Also the max speed tape is incorrect and as I climb, the airspeed stays at 320knots. It should reduce with altitude. I like the sound, but the new fonts are very difficult to read. Why change it when it worked? Also the chronometer won't work. All these features were great. Now they're gone. Can someone tell me how to increase engine power to what it was with v5. 4?
I'll add the following after I just finished a flight. 
1) No more independent control of Capt and FO ND mode - any change on one side becomes the same on the other. I' d like to see both the arc and the ILS system - this requires independent control
2) No APU condition on ECAM, In fact almost all ECAM conditions are gone
3) Icing condition no longer shown regardless of anti-ice used
4) Pre-set transition altitudes - I'd like to fill that in
5) Pre-set cost index - again, I'd like to fill that in
6) Idle power no longer is enough to taxi. Now I have to advance the throttle. 
7) No longer have strobes set to automatic - it's the pilot's job to manage this switch
8) No longer have flight phase in ECAM - ie- no lnoger have transition from take-off to crz, etc - this is part of the ECAM shortcomings per #2 above
9) Managed speeds are all incorrect per R/W A320 Neo
10) I mentioned fonts being too difficult to see. This is an awful change. Can I please revert to 5.4 settings??; I am no longer enjoying any flights since v6 was released . Who made these changes and why?
11) Throttle response is sluggish - needed power for wind shear and almost stalled waiting for it to spool up

I just reverted to 5.4 . Hopefully V0.6 will get fixed.

**Comment by tracernz** [comment, marked as answer]
Are you using the FBW A320 or Asobo A320? They are separate selections as of 0.6, see https://www.youtube.com/watch?v=HBeXOYJOAws&t=73s

**Reply by Phantomflyer81** [reply]
I messed up big time.  I understand where I went wrong and I offer my sincere apologies.

**Comment by hotshotp** [comment]
please be a little more respectful. you are using the default asobo a320neo, please choose the flybywire option instead of the asobo

**Comment by Phantomflyer81** [comment]
Had so little power using FBW 0.6. ATC kept telling me to expedite climb
but it wouldn't get to FL340 without almost stalling. And that was with
less than 10% load. So I reverted to 0.5.4

Bill Alexander

**Comment by Phantomflyer81** [comment]
Not using default. Using FBW 0.6 Sorry if you thought I was disrespectful.
Didn't mean to come across like that. I reverted to 0.5.4 because I
couldn't climb without ATC yelling at me to expedite climb.
Wish I knew how to edit these things.

Bill Alexander

**Comment by Phantomflyer81** [comment]
PS I use your installer

Bill Alexander

**Reply by Phantomflyer81** [reply]
Per above, I offer my sincere apologies. I'm an old man and got terribly confused.  I do have a question about my liveries though. There are over 200 from the flightsim dot to site and the FBW installer wants to convert them all. I don't know why all these liveries are separate and I would like to get rid of all the ones I don't really need, but I remember that many months ago, they were all in only one single folder. Before I convert any of them, can I place only the ones I want into a folder placed within my community folder after I delete the ones I no longer want? Does it matter what I name that folder? Then I will update to V0.6 and select the A320 FBW (not the Asobo -  I was using Asobo without realizing the changes required specific FBW  selection). Feeling a bit stupid here. Sorry folks...

**Comment by Phantomflyer81** [comment]
To all the people making this aircraft unbelievably realistic, I say a huge thank you. I was selecting the Asobo in error. I had 298 liveries waiting to be converted by the new installer. The amount of space they took was enormous, so I deleted them all and let the new installer update to V0.6, then fetched the only FBW dedicated 320 nx that I would use regularly; 10 in all (so far). Fired up the sim and chose the correct aircraft and Lufthansa livery. Currently flying from KIAD (3rd party bought through ORBX direct) to EDDK (also bought through ORBX Direct) and I'm extremely impressed with the work done. The FlyPad is amazing! All I can say is WOW!!!
Again, I offer my sincere apologies for being such a grump last week.
Just a few questions : if I were to offer services in the development area, how do I join and what areas are in need of assistance? What kind of expertise is required?
